fabfile/run.py

```diff
@@ -17,3 +25,63 @@ def regression():
     'Runs Citus\' regression tests'
     with cd(config.paths['citus-repo']):
         run('make check')
+
+@task
+@runs_once
+@roles('master')
+def dml_tests(*args):
+    'Runs pg-bench dml tests using the given config options'
+    dir_path = '/home/ec2-user/'
+    results_file = open(dir_path + 'dml_benchmark_results.csv', 'w')
+    config = ConfigParser.ConfigParser()
+
+    # If no argument is given, run default tests
+    # Note that you can change test sql by updating insert.sql, update.sql and delete.sql
+    if len(args) == 0:
+        config.read('default_config.ini')
+    elif len(args) == 1:
+        config.read(args)
+    else:
+        print('You should use the default config or give the name of your own config file')
+
+    for section in config.sections():
+        pg_citus_tuples = eval(config.get(section, 'postgres_citus_versions'))
+        for pg_version, citus_version in pg_citus_tuples:
+
+            results_file.write(section + ", " + pg_version + ", " + citus_version + ", ")
+
+            # create database for the given citus and pg versions
+            execute(prefix.set_prefix, dir_path + 'pg-' + pg_version + '-citus-' + citus_version)
```

Comment:
I'd move these into a separate function `SetUpPostgresForBenchmark()` or so and pass all parameters to that function so that users can easily override